### ISSUE TITLE
Select box - searching and paging

### ISSUE THREAD

**Issue by viktorsmari** [reflow-project/weloop#51]
Part of #41 

Searching `agents` (Users and Organizations)  by name is still unclear, we asked for examples in https://github.com/dyne/zenpub/issues/52

It looks like the API is **not able to filter by name**. Instead we need to be able to **paginate**, so that we are 100% sure we can see all items. Especially if we have many items, like 100+

If we have more than 15 items, we need to be able to:
- [ ] Scroll down in the list would **load** the **next 15 item** into the select box

The API returns maximum 15-30 items when asking for a page.

If we have more than 15 Items in total, it **won't show all of them** in the Select.

We need to use a **search** function in the Dropdown that queries the API, so we can find the correct user out of ALL users. <-- The API does **not** support this at the moment.
 
Imagine if we have 500 users and need to transfer material. How to find the correct user?

![2021-05-07_19-47-34](https://user-images.githubusercontent.com/1689020/117490030-95d59d00-af6e-11eb-80ac-712e96ef3f88.png)

**Comment by adam-burns** [comment]
Assuming there exists more than 15 users, this is not going to be an API issue.
This will be a client UI issue. See https://github.com/dyne/zenpub/issues/52#issuecomment-843177727
*Disclaimer* I do not know the UI framework.However, it is normal for UI widgets to manage dropdown widgets list lengths for presentation (especially for mobile screens, etc). I would *assume* that widgets like this offer configurations for scrolling and/or type-ahead list filtering to ease the user experience of long lists.

**Comment by adam-burns** [comment]
OK, after the query was posted in https://github.com/dyne/zenpub/issues/52#issuecomment-843217417 I see the limit issue likely to be a schema or server config. 

With respect to searching for names (disclaimers as a GraphQL novice), I found this in depth post [How does a GraphQL server turn a query into a response?](https://www.apollographql.com/blog/graphql-explained-5844742f195e/) useful to understand GraphQL server behaviour.

TL;dr?

This [How to search string values in GraphQL](https://stackoverflow.com/questions/37981269/how-to-search-string-values-in-graphql) answer gives a shorter explanation relevant to this issue.
